hawc/apps/assessment/actions/audit.py

```diff
@@ -239,10 +239,11 @@ def get_riskofbias_queryset(self):
         return domain_qs | metric_qs | score_qs
 
     def get_queryset(self):
-        audit_type = self.type
-        if audit_type == AuditType.EPI:
+        audit_type = self.type.value
+        if audit_type == "epi":
             audit_type = "epiv1" if self.assessment.epi_version == EpiVersion.V1 else "epiv2"
```

Comment:
Was the issue here that we were trying to grab the `value` attribute from a string instead of the enum?

Reply:
yes, we previously just grabbed the enum.value, but with this one case, we actually need another check to determine which string to use